test/e2e/functional_test.go

```diff
@@ -956,6 +956,46 @@ func (s *FunctionalSuite) TestPauseAfterAndBefore() {
 		WaitForWorkflow(fixtures.ToBeSucceeded)
 }
 
+func (s *FunctionalSuite) TestInputArtifactHttp() {
```

Comment:
So we can just run HTTP tests, please we move these to their own file? It should still have the `functional` build tag.

Reply:
yes i see! i misunderstood another comment before